static/templates/actors/familiar/sheet.hbs

```diff
@@ -0,0 +1,94 @@
+            <a class="hover-icon" data-action="show-image" data-tooltip="SIDEBAR.CharArt"><i class="fa-solid fa-image fa-fw"></i></a>
+        </div>
+
+        <div class="header-content">
+            <div class="charname">
+                <input class=" charname-value" name="name" type="text" value="{{actor.name}}"
+                    placeholder="{{localize "PF2E.NPC.Familiar"}}" spellcheck="false" />
+                <label class="familiar-title">{{localize "TYPES.Actor.familiar"}}</label>
+                {{#if data.master.id}}
+                    <label class="familiar-title"> {{data.details.level.value}}</label>
+                {{/if}}
+            </div>
+            <div class="sub-header">
+                <div class="traits">
+                    {{#each traits as |trait|}}
+                        <span data-trait="{{trait.value}}" data-tooltip="{{trait.description}}">{{localize trait.label}}</span>
+                    {{/each}}
+                    <span>{{localize size}}</span>
+                </div>
+                <div>
+                    <label for="{{document.uuid}}-master"><strong>{{localize "PF2E.Familiar.Master"}}</strong></label>
+                    <select id="{{document.uuid}}-master" name="system.master.id">
+                        {{#select master.id}}
+                            <option value="">{{localize "PF2E.Familiar.SelectMaster"}}</option>
+                            {{#if limitedSheet}}
+                                {{#if master}}<option value="{{master.id}}" selected>{{master.name}}</option>{{/if}}
+                            {{else}}
+                                {{#each masters as |eligible idx|}}
+                                    <option value="{{eligible.id}}">{{eligible.name}}</option>
+                                {{/each}}
+                            {{/if}}
+                        {{/select}}
+                    </select>
+                </div>
+            </div>
+            <section class="details-section">
+                {{#unless limitedSheet}}
+                    {{> "systems/pf2e/templates/actors/familiar/partials/header-details.hbs" }}
+                {{/unless}}
+            </section>
+        </div>
+    </header>
+    <hr>
+    <div class="main-section">
+        <div class="skills-sidebar {{#if limitedSheet}}skills-sidebar-placeholder{{/if}}">
+            <!-- SKILLS -->
+            {{#unless limitedSheet}}
+                {{> "systems/pf2e/templates/actors/familiar/partials/skills.hbs" }}
+            {{/unless}}
+        </div>
+        <div class="main-container">
+            {{#unless limitedSheet}}
+                {{> "systems/pf2e/templates/actors/familiar/partials/details.hbs" }}
+            {{/unless}}
+
+            {{!-- ABILITIES --}}
+            {{#unless limitedSheet}}
+                {{> "systems/pf2e/templates/actors/familiar/partials/abilities.hbs" }}
+            {{/unless}}
```

Comment:
Looks like all three of these can be in the same partial

Reply:
Since the second two are in the `main-container` wrapper, we'd need to go one level higher in the DOM and also pull in the effects and conditions too. But I guess that's fine right? We'd effectively have a partial for the entire bottom section of the sheet

Reply:
What if it was this, with one less partial (adding `abilities` to `details`):

```handlebars
<div class="skills-sidebar {{#if limitedSheet}}skills-sidebar-placeholder{{/if}}">
    <!-- SKILLS -->
    {{#unless limitedSheet}}
        {{> "systems/pf2e/templates/actors/familiar/partials/skills.hbs" }}
    {{/unless}}
</div>
<div class="main-container">
    {{#unless limitedSheet}}
        {{> "systems/pf2e/templates/actors/familiar/partials/details.hbs" }}
    {{/unless}}

    {{!-- CONDITIONS --}}
```

Reply:
Hope about inlining all the partials and conditionally showing them?

Reply:
Meaning something like this for each one?

```handlebars
<div class="skills-sidebar">
   {{> skills }}
</div>
...
{{#*inline "skills"}}
   {{#unless limitedSheet}}
        {{> "systems/pf2e/templates/actors/familiar/partials/skills.hbs" }}
    {{/unless}}
{{/inline}}
```

Reply:
yah! and sorry for the runaround.

Reply:
but no external file

Reply:
> but no external file

Oh yeah, sneaky! Will do, soon™!